index.js

```diff
@@ -17,15 +17,18 @@ function heightAscThanNameComparator(a, b) {
 
 /**
  * Pack a list of images with width and height into a sprite layout.
+ * options object with the following keys:
  *
  * @param   {Object[]}              imgs        Array of `{ svg: Buffer, id: String }`
```

Comment:
For the docblock, can you change it so that the `options` argument is now an `Object`, and the parameters are individually documented like `options.imgs` `options.pixelRatio`, etc..

see here for an example:
https://github.com/mapbox/whoots-js/blob/master/index.js#L12-L19